facility-all-programs.ipynb

```diff
@@ -495,19 +587,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from google.colab import files\n",
```

Comment:
Did the `import google` line get dropped somewhere?

`ModuleNotFoundError: No module named 'google'`

Reply:
I believe this will work on Google Colab, but if you're running it locally on Jupyter you may need to install the google.colab module

Reply:
e.g. this notebook doesn't `import google` but still works (in Colab) https://colab.research.google.com/github/edgi-govdata-archiving/ECHO-CWA-Compliance/blob/master/echo-cwa-compliances-inspections.ipynb

Reply:
oh ok, that makes sense